### PR TITLE
Fix for issue #4516

### PR DESCRIPTION
**Pull request by FinalAngel**
image

### CODE REVIEW

File: cms/templates/admin/cms/page/change_form.html [hunk 1]
```diff
@@ -17,6 +17,17 @@
 <script src="{% static "cms/js/dist/bundle.admin.changeform.min.js" %}" type="text/javascript"></script>
 {% endblock %}
 
+{% block content_title %}
+    {% if title %}<h1>{{ title }}</h1>{% endif %}
+    {% if not popup %}
+        <ul class="object-tools">
+            <li>
+                <a href="../{{ page.id }}/{{ language }}/preview/" target="_blank">{% trans "View on site" %}</a>
```
Comment (vxsx) [inline cms/templates/admin/cms/page/change_form.html]: is it ok to manually build the url? we don't have any url reverse helpers?

Comment (FinalAngel) [inline cms/templates/admin/cms/page/change_form.html]: I checked the history and cms, there seems to be no url for this case (it was once there like this) @yakky maybe you know?

Comment (yakky) [inline cms/templates/admin/cms/page/change_form.html]: @FinalAngel @vxsx indeed. I'll open an issue for 3.3: all the admin views are anonymous :(

**Comment by vxsx** [comment]
alright for now :+1:

**Pull request merged**